rotkehlchen/db/history_events.py

```diff
@@ -802,3 +803,50 @@ def edit_event_extra_data(
             (json.dumps(extra_data), event.identifier),
         )
         event.extra_data = extra_data
+
+    def query_wrap_stats(self, from_ts: Timestamp, to_ts: Timestamp) -> dict[str, Any]:
+        """Query simple statistics about the user to show them as a wrap for the year"""
```

Comment:
Here and in everything else can you specify in the docstrings this is a temporary endpoint?